docs/docs/en/development/api-test.md

```diff
@@ -0,0 +1,94 @@
+# DolphinScheduler API Automation Test
+
+## I. Preparatory knowledge
+
+### 1. The difference between API Test and Unit Test
```

Comment:
we should avoid using the number in markdown title, because the leading number `#` already represented this 
```suggestion
### The difference between API Test and Unit Test
```

Reply:
I'll fix it.

Reply:
Done.